networks/2_get_network_communities.py

```diff
@@ -64,40 +57,44 @@
 # A cluster is a set of core samples that can be built by recursively taking a core sample, finding all of its neighbors that are core samples, finding all of their neighbors that are core samples, and so on. A cluster also has a set of non-core samples, which are samples that are neighbors of a core sample in the cluster but are not themselves core samples. Intuitively, these samples are on the fringes of a cluster.
 #
 # * We define a core sample as being a sample in the dataset such that there exist `min_samples` other samples within a distance of `eps`, which are defined as neighbors of the core sample.
-# * Using all default values
+# * Here we use `eps=8` based on the observations in the [prevous notebook](1_correlation_analysis.ipynb)
```

Comment:
I'm not sure if I found these analyses/conclusions in `1_correlation_analysis.ipynb`. Are you inferring this value (`eps=8`) from the distribution plots?

Reply:
Yes I'm using the distribution plots for pdist to determine this `eps` value. I will update the comment to make this more clear though